src/Microsoft.AspNetCore.Diagnostics/DeveloperExceptionPage/Views/StackFrameInfo.cs

```diff
@@ -1,54 +0,0 @@
-// Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
I'll leave this type in. We should've put all the view generation code in an internal namespace - none of it is very useful.